TramsDataApi/DatabaseModels/A2BApplication.cs

```diff
@@ -7,38 +8,45 @@ namespace TramsDataApi.DatabaseModels
     [Table("A2BApplication", Schema="sdd")]
     public class A2BApplication
     {
-        [Key]
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-        public int ApplicationId {get; set;}
-        public string Name {get; set;} 
-        public string ApplicationType {get; set;}
-        public string FormTrustProposedNameOfTrust {get; set;}
-        public bool ApplicationSubmitted {get; set;}
-        public string ApplicationLeadAuthorId {get; set;}
-        public string ApplicationVersion {get; set;}
-        public string ApplicationLeadAuthorName {get; set;}
-        public string ApplicationRole {get; set;}
-        public string ApplicationRoleOtherDescription {get; set;}
-        public int? ChangesToTrust {get; set;}
-        public string ChangesToTrustExplained {get; set;}
-        public int? ChangesToLaGovernance {get; set;}
-        public string ChangesToLaGovernanceExplained {get; set;}
-        public DateTime? FormTrustOpeningDate {get; set;}
-        public string TrustApproverName {get; set;}
-        public string TrustApproverEmail {get; set;}
-        public string TrustId { get; set; }
-        public int? FormTrustReasonApprovalToConvertAsSat {get; set;}
-        public string FormTrustReasonApprovedPerson {get; set;}
-        public string FormTrustReasonForming {get; set;}
-        public string FormTrustReasonVision {get; set;}
-        public string FormTrustReasonGeoAreas {get; set;}
-        public string FormTrustReasonFreedom {get; set;}
-        public string FormTrustReasonImproveTeaching {get; set;}
-        public string FormTrustPlanForGrowth {get; set;}
-        public string FormTrustPlansForNoGrowth {get; set;}
-        public int? FormTrustGrowthPlansYesNo {get; set;}
-        public string FormTrustImprovementSupport {get; set;}
-        public string FormTrustImprovementStrategy {get; set;}
-        public string FormTrustImprovementApprovedSponsor {get; set;}
+	    public string Name { get; set; }
+	    
+	    [Key]
+	    public string ApplicationId { get; set; }
```

Comment:
Remind me, why are we changing this to a string? I know it's a string in Dynamics but if we're moving away from dynamics, wouldn't we remodel this as an int, seeing as the actual data in it is an int.

Reply:
I was told that the the application is a string not an int value.  I had it as int value initially but was told that it would come through from a2b external code as a string value that couldn't be translated to an string ie 'APP_12234'